website/content/en/blog/product-thinking-for-platforms.md

```diff
@@ -0,0 +1,134 @@
+
+
+### Continuous Approach
+
+The Product Mindset differs from the traditional project-oriented management approach. While a project is usually focused around larger phases of requirements gathering, implementation and deployment, product thinking is meant to operate continuously. When using product thinking, the goal is to stay close to the customer, figure out their needs and deliver value to them in a continuous cycle. By engaging with customers continuously you significantly raise the odds of delivering value for them. And when you build a track record of delivering value, you build trust with customers and they keep coming back to you with new opportunities. Contrast this with the project oriented approach where a large project that take huge amounts of time and money to implement misses the mark with the underlying need. In this case everyone usually wants to forget this happened and you end up with another partially implemented solution that becomes a drag.
+
+This mindset shift from project to product is essential for creating successful platforms.
+
+
+## How does Product Thinking fit with platforms?
+
+Modern platforms and platform engineering are focused on reducing cognitive load for engineers and allowing them to focus on creating business value. It only makes sense to treat the platform as a product that becomes part of the value chain of the business. 
+
+Platforms are for building things and running them. A platform user needs to be able to do both.
+
+
+### Find the platform user and value
+
+If we accept this premise (and we'll get into why you would want to below) then we need to figure out how to create value for our users. As we mentioned above, our users are creating value for the business and anything we can do to aid in that process will be of value to them.
+
+This is where the rubber hits the road with product thinking and platform engineering. 
+
+You have two choices when you build platforms for people to use: you can be focused on their problems or your own. This might seem counter-intuitive. Of course when you build a platform it is for other people to use, right? Well, maybe. In the infrastructure world, we frequently build platforms for ourselves to deal with inbound requests from our users. The approach to automation is a good example. Much traditional automation is focused on dealing with either BAU infrastructure work or automating our response to inbound user requests. 
```

Comment:
```suggestion
You have two choices when you build platforms for people to use: you can be focused on their problems or your own. This might seem counter-intuitive. Of course when you build a platform it is for other people to use, right? Well, maybe. In the infrastructure world, we frequently build platforms for ourselves to deal with inbound requests from our users. The approach to automation is a good example. Much traditional automation is focused on dealing with either business-as-usual infrastructure work or automating our response to inbound user requests. 
```

Didn't really know the BAU word yet :)

Reply:
Replaced with day to day.